internal/flags/info.go

```diff
@@ -6,16 +6,18 @@ import (
 )
 
 type AmtInfoFlags struct {
-	Ver      bool
-	Bld      bool
-	Sku      bool
-	UUID     bool
-	Mode     bool
-	DNS      bool
-	Cert     bool
-	Ras      bool
-	Lan      bool
-	Hostname bool
+	Ver        bool
+	Bld        bool
+	Sku        bool
+	UUID       bool
+	Mode       bool
+	DNS        bool
+	Cert       bool
+	UserCert   bool
```

Comment:
Please remove this user and system cert as it belongs to different feature changes